statistik/controller.py

```diff
@@ -597,40 +659,53 @@ def make_nav_links(level=None, style='SP', version=None, user=None, elo=None,
     :param int clear_type:  Rating type (refer to Chart model for options)
     :rtype list:            List of tuples of format (link text, link)
     """
-    ret = [(_('INDEX'), reverse('index')),
-           (_('SEARCH'), reverse('search'))]
+    ret = [(_('INDEX'), reverse('index', kwargs={'game': GAME_CHOICES[game][1]})),
+           (_('SEARCH'), reverse('search', kwargs={'game': GAME_CHOICES[game][1]}))]
```

Comment:
pls at least set `GAME_CHOICES[game][1]` to a name and use it throughout method, this applies elsewhere in this PR